### PR TITLE
Homepage grad panel update

### PR DESCRIPTION
**Pull request by waidelacruz**
Updated grad panel, and adjusted other homepage sections to match.

### CODE REVIEW

File: app/views/widgets/home_page/_chapman_family.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <section id="chapmanFamily" class="section light-bg stories">
   <div class="maxWidth">
-    <h2 class="heading">One Chapman Family. Many #ChapmanStories.</h2>
+    <h2 class="heading" skrollable skrollable-after" data-bottom="opacity:0;" data-center="opacity:1;">One Chapman Family. Many #ChapmanStories.</h2>
```
Comment (tatwell) [inline app/views/widgets/home_page/_chapman_family.html]: It looks like there's an extra quotation mark here.

File: app/views/widgets/home_page/_general_info.html [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <section class="section light-bg" id="generalInformation">
   <div class="maxWidth">
-    <h2 class="heading">At Chapman, you'll discover there's an</h2>
-    <h3 class="subheading" data--150-bottom="opacity:1;" data--150-bottom-top="opacity:0;">energy that runs through everything and everyone.</h3>
+    <h2 class="heading" skrollable skrollable-after" data-bottom="opacity:0;" data-center="opacity:1;">At Chapman, you'll discover there's an energy that runs through everything and everyone.</h2>
```
Comment (tatwell) [inline app/views/widgets/home_page/_general_info.html]: Looks like another extra quotation mark. Slightly surprised this isn't breaking.

Comment (waidelacruz) [inline app/views/widgets/home_page/_general_info.html]: Fixed.

Comment (waidelacruz) [inline app/views/widgets/home_page/_general_info.html]: Fixed the second quote.

File: app/views/widgets/home_page/_chapman_family.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <section id="chapmanFamily" class="section light-bg stories">
   <div class="maxWidth">
-    <h2 class="heading" skrollable skrollable-after" data-bottom="opacity:0;" data-center="opacity:1;">One Chapman Family. Many #ChapmanStories.</h2>
+    <h2 class="heading" skrollable skrollable-after data-bottom="opacity:0;" data-center="opacity:1;">One Chapman Family. Many #ChapmanStories.</h2>
```
Comment (tatwell) [inline app/views/widgets/home_page/_chapman_family.html]: I think the line should be this:

     <h2 class="heading skrollable skrollable-after" data-bottom="opacity:0;" data-center="opacity:1;">

File: app/views/widgets/home_page/_general_info.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <section class="section light-bg" id="generalInformation">
   <div class="maxWidth">
-    <h2 class="heading" skrollable skrollable-after" data-bottom="opacity:0;" data-center="opacity:1;">At Chapman, you'll discover there's an energy that runs through everything and everyone.</h2>
+    <h2 class="heading" skrollable skrollable-after data-bottom="opacity:0;" data-center="opacity:1;">At Chapman, you'll discover there's an energy that runs through everything and everyone.</h2>
```
Comment (tatwell) [inline app/views/widgets/home_page/_general_info.html]: Same as last comment. `skrollable skrollable-after` are class names if I'm not mistaken.

Comment (waidelacruz) [inline app/views/widgets/home_page/_general_info.html]: Whoops. Corrected.

**Pull request merged**